operators/config/samples/elasticsearch/elasticsearch.yaml

```diff
@@ -4,11 +4,15 @@ kind: Elasticsearch
 metadata:
   name: elasticsearch-sample
 spec:
-  version: "7.1.0"
+  version: "7.2.0"
```

Comment:
Should we just go all the way to 7.2.1?

Reply:
👍

Reply:
Let's do it for all docs actually, including quickstart.
I'll open another PR.